src/main/java/com/jcabi/github/mock/MkReleases.java

```diff
@@ -176,4 +205,37 @@ private String xpath() {
             this.coords
         );
     }
+
+    /**
+     * Check the first release is greater than last release.
+     * @param first First release.
```

Comment:
@weisebrazil remove dot's after the `@param`/`@return` description (dot's should be only in the method/constructor description)